custom-elements/custom-elements-registry/define.html

```diff
@@ -0,0 +1,178 @@
+<!DOCTYPE html>
+<title>Custom Elements: Element definition</title>
+<link rel="help" href="https://html.spec.whatwg.org/multipage/scripting.html#element-definition">
+<meta name="author" title="Koji Ishii" href="mailto:kojiishi@gmail.com">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<body>
+<div id="log"></div>
+<script>
+'use strict';
+(function () {
+  var testable = false;
+  test(function () {
+    assert_true('customElements' in window, '"window.customElements" exists');
+    assert_true('define' in window.customElements, '"window.customElements.define" exists');
+    testable = true;
+  }, '"window.customElements.define" exists');
+  if (!testable)
+    return;
+
+  // 1. If IsConstructor(constructor) is false,
+  // then throw a TypeError and abort these steps.
+  test(function () {
+    assert_throws(new TypeError, function () {
+      window.customElements.define();
+    });
+  }, 'If no arguments, throw a TypeError');
+  test(function () {
+    assert_throws(new TypeError, function () {
+      window.customElements.define("test-define-one-arg");
+    });
+  }, 'If one argument, throw a TypeError');
+  [
+    [ "undefined", undefined ],
+    [ "null", null ],
+    [ "object", {} ],
+    [ "string", "string" ],
+    [ "function", function () {} ],
```

Comment:
As noted in https://github.com/w3c/web-platform-tests/pull/2940#issuecomment-216559093 this should be replaced